Shared/Samples/Augment reality to navigate route/AugmentRealityToNavigateRouteView.swift

```
@@ -0,0 +1,253 @@
+        return elevationSurface
+    }()
+    /// The elevation source with elevation service URL.
+    @State private var elevationSource = ArcGISTiledElevationSource(url: URL(string: "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer")!)
+    /// The graphics overlay containing a graphic.
+    @State private var graphicsOverlay = GraphicsOverlay()
+    /// The status text displayed to the user.
+    @State private var statusText = "Adjust calibration before starting."
+    /// A Boolean value indicating whether the use is navigatig the route.
+    @State private var isNavigating = false
+    /// The result of the route selected in the route planner view.
+    @State private var routeResult: RouteResult?
+    
+    init() {
+        elevationSurface.addElevationSource(elevationSource)
+        scene.baseSurface = elevationSurface
+    }
+    
+    var body: some View {
+        if isShowingRoutePlanner {
+            RoutePlannerView(isShowing: $isShowingRoutePlanner)
+                .onDidSelectRoute { routeGraphic, routeResult  in
+                    self.routeResult = routeResult
+                    graphicsOverlay = makeRouteOverlay(
+                        routeResult: routeResult,
+                        routeGraphic: routeGraphic
+                    )
+                }
+                .task {
+                    Task { try await elevationSource.load() }
+                }
+        } else {
+            WorldScaleSceneView { _ in
+                SceneView(scene: scene, graphicsOverlays: [graphicsOverlay])
+            }
+            .calibrationButtonAlignment(.bottomLeading)
+            .task {
+                Task {
+                    try await locationDataSource.start()
+                    
+                    for await location in locationDataSource.locations {
+                        try await routeDataModel.routeTracker?.track(location)
+                    }
+                }
```

`Task` shouldn't be used just to discard an error.

```suggestion
                try? await locationDataSource.start()
                
                for await location in locationDataSource.locations {
                    try? await routeDataModel.routeTracker?.track(location)
                }
```